cmd/ooniprobe/internal/ooni/ooni.go

```diff
@@ -32,7 +32,8 @@ var logger = log.WithFields(log.Fields{
 // ProbeCLI is the OONI Probe CLI context.
 type ProbeCLI interface {
 	Config() *config.Config
-	DB() *database.Database
+	WriteDB() model.WritableDatabase
+	ReadDB() model.ReadableDatabase
```

Comment:
Same as above (i.e., code in `cmd/ooniprobe` can continue to use a concrete database)